### PR TITLE
Update reusable-doc-test.yaml for bug fix in geoips#840 and geoips#850

### PR DESCRIPTION
**Pull request by biosafetylvl5**
Fixes call argument for build_docs.py (from `--force FORCE` to just `--force`)

### CODE REVIEW

**Comment by biosafetylvl5** [comment]
Wait until #840 and #850 on geoips are merged for this fix

**Review by jsolbrig — APPROVED** [review]

**Comment by jsolbrig** [comment]
I approved this with the failing tests because those are expected to fail for the time being. We do need to figure out how to handle those tests which depend on the doclinttest image from the geoips repo, but that is a different issue for a different PR.

We should merge this so we can get the CI working prior to merging geoips#850 and geoips#840.

**Review by mindyls — APPROVED** [review]

**Pull request merged**
